stylus/components/overlay.styl

```diff
@@ -0,0 +1,13 @@
+@require '../settings/palette'
```

Comment:
Just wondering, why separate this from the modal styles. What's the gain?

Reply:
The gain is that we have small components in charge of only 1 thing. Here I have separated the Overlay in charge of blocking the scroll and reacting to onEscape from the modal since the overlay can be used in other situations, for example an action menu. Having a separate style communicates this separation and is in line with the separation of the JS files.

Reply:
Small components in charge of only one thing is another name for "Do one thing and do it well", the Unix philosophy. This enables composition, separation of concerns, DRYness and is generally a good way of doing things.

Reply:
OK then,  I thought that overlays were used only with modals so that they were strictly part of it.